src/usecases/sendBookingNotification.js

```diff
@@ -69,4 +78,21 @@ const sendBookingNotification = ({
   }
 };
 
+const getTemplateIds = (notificationType) => {
+  let textMessageTemplateId;
+  let emailTemplateId;
+
+  if (notificationType == NEW_NOTIFICATION) {
+    textMessageTemplateId = TemplateStore().firstText.templateId;
+    emailTemplateId = TemplateStore().firstEmail.templateId;
+  }
+
+  if (notificationType == UPDATED_NOTIFICATION) {
+    textMessageTemplateId = TemplateStore().updatedVisitText.templateId;
+    emailTemplateId = TemplateStore().updatedVisitEmail.templateId;
+  }
```

Comment:
How about converting this to a switch statement, then throwing an exception on the default case for when it's passed something not recognised?

Reply:
Yep good idea, that keeps all the notification type logic in one place